mrp_bom_attribute_match/reports/mrp_report_bom_structure.py

```diff
@@ -43,3 +46,67 @@ def _get_bom_lines(self, bom, bom_quantity, product, line_id, level):
                     if isinstance(value, models.NewId):
                         component[key] = value.origin
         return components, total
+
+    def _get_price(self, bom, factor, product):
```

Comment:
Would it be possible to add a few inline comments to indicate the patched lines?
As this is another complete override, it's always useful to know exactly what changed, specially for migrations.

Reply:
Agree